python/uv/_build_backend.py

```diff
@@ -37,13 +37,13 @@ def call(args: "list[str]", config_settings: "dict | None" = None) -> str:
     if result.returncode != 0:
         sys.exit(result.returncode)
     # If there was extra stdout, forward it (there should not be extra stdout)
-    result = result.stdout.decode("utf-8").strip().splitlines(keepends=True)
-    sys.stdout.writelines(result[:-1])
+    sdtout = result.stdout.decode("utf-8").strip().splitlines(keepends=True)
```

Comment:
typo: should be spelled `stdout`, not `sdtout`

Reply:
🤦‍♂️